.github/workflows/ee-nlc-tag-push.yml

```diff
@@ -99,6 +104,7 @@ jobs:
         run: |
           docker buildx build --push \
             --build-arg HZ_VERSION=${HZ_VERSION} \
+            --build-arg JDK_VERSION=${{ matrix.jdk }} \
             --build-arg HAZELCAST_ZIP_URL=${HAZELCAST_ZIP_URL} \
             --tag ${NLC_IMAGE_NAME}:${HZ_VERSION}  \
```

Comment:
You need modify tags to have `${NLC_IMAGE_NAME}-jdk${{ matrix.jdk }}:${HZ_VERSION}` for all JDK versions and additionally `${NLC_IMAGE_NAME}-:${HZ_VERSION}` for the default JDK